### DISCUSSION TITLE
which template is this?

### DISCUSSION THREAD

**Discussion by osbyrne** [unocss/unocss#3944] [Q&A] [answered]
I was looking at the unocss landing page and it looks very similar to many others (rollup, vite, shiki, astria, vitepress, wagmi...)
look at this: https://x.com/unlikelydoorway/status/1785823845762384243

Can anyone find which "template" these websites are using? 

![image](https://github.com/unocss/unocss/assets/68084397/5ab24699-13b7-47b1-af2b-2b6e1ae32bc4)
![image](https://github.com/unocss/unocss/assets/68084397/83592555-bef2-4dcf-bafb-01cad1334837)
![image](https://github.com/unocss/unocss/assets/68084397/19333ece-caa7-4785-ae0d-24e5b434f1f2)
![image](https://github.com/unocss/unocss/assets/68084397/06fb8c5a-00fa-4cb0-96c4-66ac042173a9)
![image](https://github.com/unocss/unocss/assets/68084397/0adf457e-ad02-48ce-827e-83a3d6db8d3b)
![image](https://github.com/unocss/unocss/assets/68084397/d322fd6c-9324-4be0-adc1-e8876182bfba)
![image](https://github.com/unocss/unocss/assets/68084397/c23e59eb-c26f-48db-a5be-b2d09c93e052)
![image](https://github.com/unocss/unocss/assets/68084397/cad3d238-fa29-499c-bc2a-b1eba4126ebb)

**Comment by userquin** [comment, marked as answer]
VitePress: https://github.com/vuejs/vitepress

You can find the docs in the docs folder in this repo: https://github.com/unocss/unocss/tree/main/docs
